bin/run-tc-ru_transfer.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+set -xe
+
+ru_dir="./data/ru"
+ru_csv="${ru_dir}/ru.csv"
+
+epoch_count=$1
+
+if [ ! -f "${ru_dir}/ru.csv" ]; then
+    echo "Downloading and preprocessing Russian example data, saving in ${ru_dir}."
+    python -u bin/import_ru.py ${ru_dir}
+fi;
+
+# Force only one visible device because we have a single-sample dataset
+# and when trying to run on multiple devices (like GPUs), this will break
+export CUDA_VISIBLE_DEVICES=0
+
+# WORKS: #
+
+echo "# CUDNN FINE"
```

Comment:
Could we have a nit more descriptive messages here ?

Reply:
fixed